bind/gencpy.go

```diff
@@ -259,6 +260,13 @@ func (g *cpyGen) gen() error {
 	g.impl.Indent()
 	g.impl.Printf("PyObject *module = NULL;\n\n")
 
+	version := runtime.Version()
```

Comment:
instead of having this check being made during the `gencpy` step, put it during the `gengo` step.